sr_grasp/src/sr_grasp/__init__.py

```diff
@@ -122,10 +125,14 @@ def get_grasp_array(self):
         arr.grasps = self.get_all()
         return arr
 
-    def get_grasp(self, id):
+    def get_grasp(self, grasp_index):
         """Return a single grasp from the stash from it's id field."""
-        Grasp()
-        return Grasp;
+        grasp = None
+        try:
+            grasp = self._store[grasp_index]
+        except IndexError:
```

Comment:
self._store is a dictionary as far as I can see. The exception for incorrect key is KeyError not IndexError. Also for dictionary I would suggest to use method get which has default value in case if key doesn't exists in dictionary.